platform/admin/core/admin-core-insecuredefaults/src/main/java/org/codice/ddf/admin/insecure/defaults/service/InsecureDefaultsService.java

```diff
@@ -13,26 +13,20 @@
  */
 package org.codice.ddf.admin.insecure.defaults.service;
 
-import java.lang.management.ManagementFactory;
 import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.List;
 
-import javax.management.InstanceAlreadyExistsException;
-import javax.management.InstanceNotFoundException;
-import javax.management.MBeanRegistrationException;
-import javax.management.MBeanServer;
-import javax.management.MalformedObjectNameException;
-import javax.management.NotCompliantMBeanException;
-import javax.management.ObjectName;
+import javax.annotation.Nonnull;
 
 import org.codice.ddf.configuration.AbsolutePathResolver;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class InsecureDefaultsServiceBean implements InsecureDefaultsServiceBeanMBean {
```

Comment:
The mbean registration in this class was removed and is now part of `AlertServiceBean` instead.